x-pack/plugins/cloud_security_posture/public/pages/findings/use_findings_by_resource.ts

```diff
@@ -51,7 +64,8 @@ export const getFindingsByResourceAggQuery = ({
     aggs: {
       groupBy: {
         composite: {
-          size: 10 * 1000,
+          size,
+          after,
```

Comment:
pagination for [composite aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-composite-aggregation.html)